src/renderer/zbay/messages.js

```diff
@@ -397,7 +395,7 @@ export const createTransfer = (values) => {
     sender: values.sender,
     receiver: values.receiver,
     createdAt: DateTime.utc().toSeconds(),
-    memo: memo || '',
+    memo: memo || 'TRASNFER',
```

Comment:
typo: 'TRANSFER'

also, it's not your change right? it's a change coming from upstream, from Janusz?